tests/test_amgx_direct_solver.cu

```diff
@@ -190,6 +191,7 @@ BOOST_AUTO_TEST_CASE(amgx_2_procs)
     mfmg::CudaSolver<dealii::LinearAlgebra::distributed::Vector<
         double, dealii::MemorySpace::CUDA>>
         direct_solver_dev(cuda_handle, cuda_op, params);
+    direct_solver_dev.apply(rhs_dev, solution_dev);
```

Comment:
Weren't we getting warnings  about not using the solver?!

Reply:
No :(  I have seen that several times when it's a cuda file. I don't know if it's because of nvcc or if we are not passing the flags correctly.